nipype/interfaces/base.py

```diff
@@ -1505,18 +1518,18 @@ class must be instantiated with a command argument
     >>> from nipype.interfaces.base import CommandLine
     >>> cli = CommandLine(command='ls', environ={'DISPLAY': ':1'})
     >>> cli.inputs.args = '-al'
-    >>> cli.cmdline # doctest: +ALLOW_UNICODE
+    >>> cli.cmdline
     'ls -al'
 
     # Use get_traitsfree() to check all inputs set
-    >>> pprint.pprint(cli.inputs.get_traitsfree())  # doctest: +NORMALIZE_WHITESPACE +ALLOW_UNICODE
```

Comment:
we should add these tags back in?
```
Expected:
    {'args': '-al',
     'environ': {'DISPLAY': ':1'},
     'ignore_exception': False}
Got:
    {'args': '-al', 'environ': {'DISPLAY': ':1'}, 'ignore_exception': False}
```

Reply:
the same here

Reply:
Seems like `+ALLOW_UNICODE` did work right?

Reply:
@oesteban @mgxd - so the way how Mathias is testing than I always do:
```
conda create -n testnipype python=2 pip
source activate testnipype
conda install -y nipype
pip install -U https://github.com/nipy/nipype/archive/master.zip
python -c "import nipype; nipype.test()"
```
Indeed, if you test like this, pytest is testing nipype installed in the environment, i.e. `....../envs/testnipype/lib/python2.7/site-packages/nipype/interfaces/base.py` and there is no `pytest.ini` to collect.

Reply:
I'm just wondering if the `pytest.ini` shouldn't be in`nipype/nipype`. any thoughts?